schema/xsd/common.xsd

```diff
@@ -101,4 +101,12 @@
         </xs:restriction>
     </xs:simpleType>
 
+    <xs:simpleType name="identifier-in-signed-documents">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="PERSONAL_IDENTIFICATION_NUMBER_AND_NAME"/>
+            <xs:enumeration value="DATE_OF_BIRTH_AND_NAME"/>
+            <xs:enumeration value="NAME"/>
```

Comment:
Lurer på om jeg synes det hadde vært litt bedre med
```
PERSONAL_IDENTIFICATION_NUMBER
DATE_OF_BIRTH
NAME_ONLY
```

Så kunne vi dokumentert at navnet fulgte med på de andre også. Men kode > dokumentasjon, I guess? Jeg bare misliker så lange navn på konstantene 😩

Reply:
Jeg er enig i at konstantene har litt klønete lange navn, men liker at de er selvdokumenterene.  Det som gjør det hele litt kilent er selvfølgelig at autentiserte esignaturer hvor avsender ønsker fødselsnummer ikke _alltid_ inneholder navn (hvis personen ikke kan slås opp i DSF eller hvis DSF-tjenesten vi bruker er utilgjengelig ved signeringstidspunktet), så `PERSONAL_IDENTIFICATION_NUMBER_AND_NAME` er ikke garantert _alltid sant_.

Kanskje det er bedre å bruke navnene du foreslår, og slenge inn et par setninger i dokumentasjonen. Det er ikke _så_ vanskelig å skrive dokumentasjon lissom 🤔

Reply:
Jeg er helt uenig her. Jeg vil ha _AND_NAME_ med fordi da sier det hva som inneholdes i stringen i normaltilfeller. At oppslag feiler i noen tilfeller er feilhåndtering, og er i mine 👀  helt ok. Sååå lang er ikke de konstantene der, og vi sitter i et IDE med autocomplete. I en signeringsverden der det blir flere og flere caser så vil jeg veeeeeldig gjerne at konstantene sier så mye som mulig. 😁

Reply:
Altsååå: helt uenig med @fredva, litt uenig med @SindreNordbo om vi endrer til å ikke ha med navn. heh. Uklar fyr.

Reply:
Helt fint for meg det, jeg har ikke noen veldig sterke følelser hverken den ene eller den andre veien. Kode > dokumentasjon, så da stiller jeg meg bak argumentene til @asjafjell og stemmer for å la det være slik det er nå :)

Reply:
Han overbeviste meg.